python/paddle/distributed/fleet/utils/ps_util.py

```diff
@@ -13,11 +13,20 @@
 # limitations under the License.
 """Parameter Server utils"""
 
+from __future__ import annotations
+
 import os
 import warnings
+from typing import TYPE_CHECKING
 
 import paddle
 
+if TYPE_CHECKING:
+    from paddle import Tensor
+    from paddle.base import Executor
+    from paddle.distributed.fleet.base.role_maker import RoleMakerBase
+    from paddle.framework import Program
```

Comment:
```suggestion
    from paddle.static import Program
```

`paddle.framework.Program` 不是公开 API，而且运行时有点细微差异，比如说 PIR 模式下并没有给 `paddle.framework.Program` patch 成 PIR 的 Program，虽然 patch 这种行为本身类型检查工具也不可能检查出来就是了……